index.html

```diff
@@ -81,11 +81,12 @@ <h1><a rel="home" title="Home" href="index.html" class="floe-header-logo">floe
                         </h1>
                     </header>
                 </div>
-                <div class="columns small-12 medium-3 large-3 align-self-bottom">
+                <div class="columns small-12 medium-12 large-5 align-self-bottom">
                     <nav class="floe-nav">
                         <ul>
-                            <li><a href="news/index.html">News</a></li>
-                            <li><a href="resources.html">Resources</a></li>
+                            <li><a href="./news/index.html">News</a></li>
+                            <li><a href="./resources.html">Resources</a></li>
+                            <li><a href="./ui-options.html">UI Options</a></li>
```

Comment:
It's a bit harder to visually distinguish these as nav elements and that "UI Options" is an atomic unit. It's not terrible because the spacing between "UI" and "Options" is smaller than the spacing between nav links. Do you think there is a quick styling change we can apply to better identify these, or should we just wait for a site redesign, and/or file this as a separate issue?

Reply:
Actually looking at the screenshots here doesn't look too bad, it might that I'm just focusing on it. Anyways, curious to know your thoughts.

Reply:
I agree it can be better, but it's not unusable at least. Some visual separation would be useful but perhaps best left for a redesign.